api/app/utils/daily_template.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
```

Comment:
I think this should be somewhere else. I don't think a daily_template.html file is a util? Beter suited in app.hfi, or a general template folder?

Reply:
Sure, will move it to app.hfi